MIPs/mip-mc-handshake.md

```diff
@@ -0,0 +1,255 @@
+
+**Scope**: A uniquely identified domain for which authorizations can be applied (see [CAIP-217](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-217.md) for further definition).
+
+**Authorization**: A permission to use a resource. This term may be used interchangeably with the term **Permission**.
+
+**eip155 scope**: A scope that applies to a particular EVM-compatible chain identified as specified in [CAIP-2](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-2.md).
+
+**wallet scope**: A scope that applies to capabilities that a MetaMask Wallet client can handle in a relatively self-contained way.
+
+**chain**: A distributed ledger that can be targeted. It may or may not refer to a blockchain, which is a type of distributed ledger.  
+
+**network**: An RPC network or node that enables interactions with a chain.
+
+
+## Proposal Specification
+
+### Implementing `provider_authorize`
+
+- While `provider_authorize` will not be available through an EIP-1193 provider, the Multichain API Delivery MIP provides details about how to access the multichain API. 
+
+- Authorization requests for both `wallet` and chain-specific scopes SHOULD generally be placed into `optionalScopes`. Use of `requiredScopes` will be supported but is NOT RECOMMENDED
+
+- `chainId` uniquely indentifies the target chain, formatted as a string that includes both the namespace and the reference, following CAIP-2 conventions.
+  
+- `methods`, `notifications`, and `rpcEndpoints` are listed directly under each scope in line with CAIP-25.
+
+- Additional metadata fields in `rpcEndpoints` SHOULD be compliant with the fields specified in EIP-3085.
+
+[ TODO a PR in the api-spec repo describing the provider_authorize method in OpenRPC should be added here ]
+
+#### Valid Scopes
+Valid scopes SHALL include and will initially be limited to:
+- `wallet`
+- `eip155:X` (eip155 scope)
+
+> **Note:** In the future, this specification may be extended to support Snap IDs or the full range of [CAIP-2 Namespaces](https://namespaces.chainagnostic.org/)
+
+#### Wallet-Scope
+##### Valid methods
+See MetaMask’s rpcDocument for the most up-to-date specification of supported methods: https://metamask.github.io/api-specs/latest/openrpc.json
+
+`wallet_switchEthereumChain` method WILL NOT be carried over to the Multichain API as chain switching coordination between dApp and wallet will no longer be necessary
+`eth_sign`, `eth_signTypedData`, `eth_signTypedData_v3` methods WILL NOT be included in the Multichain API as these signature methods have been superseded by the more commonly used `personal_sign` and `eth_signTypedData_v4` methods
+
+##### Valid notifications
+`accountsChanged` - Because `provider_authorize` will return the accounts that the user authorized a dapp to interact with in the response, this notification is no longer necessary. The `accountsChanged` notification WILL NOT be included in the Multichain API.
+
+#### eip155:X Scope
```

Comment:
`eip155` isnt a great name for this namespace since smart contract accounts exist and cannot sign via eip155. `ethereum` or `evm` is more inline with the other namespaces they have already like `cosmos`, `casper`, etc.

Reply:
155 isn't signing, it's the chainId system! there was definitely a lot of discussion back and forth on this, and the main argument against "EVM" was that there is no technical definition of EVM (much to the delight of L2 CMOs everywhere, hehe).  the main argument in favor of eip155 is that it CAIP schemes are all heirarchical, and "eip155" explains the next layer in the matruyshka doll :D 

some partial context [here](https://github.com/ChainAgnostic/namespaces/pull/2/files) (CASA discord has more if you dig into the older channels)